js/bitmart.js

```diff
@@ -2769,6 +2770,96 @@ module.exports = class bitmart extends Exchange {
         };
     }
 
+    async fetchBorrowRates (params = {}) {
+        /**
+         * @method
+         * @name bitmart#fetchBorrowRates
+         * @description fetch the borrow interest rates of all currencies
+         * @see https://developer-pro.bitmart.com/en/spot/#get-trading-pair-borrowing-rate-and-amount
+         * @param {dict} params extra parameters specific to the bitmart api endpoint
+         * @returns {dict} a list of [borrow rate structures]{@link https://docs.ccxt.com/en/latest/manual.html#borrow-rate-structure}
+         */
+        await this.loadMarkets ();
+        const request = {};
+        const response = await this.privateGetSpotV1MarginIsolatedPairs (this.extend (request, params));
```

Comment:
Is there a way to get the cross borrow rates as well? Or are they any different?

Reply:
Bitmart currently only supports isolated margin